### DISCUSSION TITLE
Support for Nextjs Turbopack

### DISCUSSION THREAD

**Discussion by onimitch** [tamagui/tamagui#3015] [General]
Although turbopack is still marked as experimental, it seems it's reached an (aparantly) stableish state for development use, passing 100% of it's unit tests https://areweturboyet.com/

Docs: https://nextjs.org/docs/architecture/turbopack
Config: https://nextjs.org/docs/app/api-reference/next-config-js/turbo

Are there any plans to support Turbopack with the Tamagui compiler?

It seems that Turbopack replaces Webpack, so from what I understand it will require a rewrite of the current Tamagui webpack plugin. It sounds like a decent chunk of work so I'm not expecting anything any time soon, but just wanted to raise it anyway.

**Comment by natew** [comment]
We have a way to build your current app just using the tamagui CLI. We need to document that though @jonsherrard this would be a nice add-on to docs and testing just to make sure it works. See `@tamagui/cli` build command.

**Comment by natew** [comment]
If anyone knows how to write a turbopack plugin I can start on this work soon. Would love to know but docs seem sparse here.
